lib/CL/devices/common.h

```diff
@@ -26,7 +26,7 @@
 #define POCL_COMMON_H
 
 #include "pocl_cl.h"
-#include "dev_image.h"
+#include "_kernel_c.h"
```

Comment:
I believe cl_int is not defined in kernel context, it's simply int there, I think.